ComputerVisionServer/moedx/object_detection/object_detectron2.py

```diff
@@ -42,11 +42,8 @@
 # create predictor
 predictor = DefaultPredictor(cfg)
 
+# TODO: Programmatically determine if GPU support exists.
```

Comment:
Soon?

Reply:
This "detectron2" version is not in use. Keeping the source code around in case we ever decide to use it.